backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5655120_sys_replace_PriceListVersion_pricing_rule.sql

```diff
@@ -0,0 +1,5 @@
+-- 2022-09-07T14:37:36.691Z
```

Comment:
looking at `5655670_sys_PriceListVersionPricingRule_Classname.sql` i guess we can entirely drop this one, right?

Reply:
refactored